addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -337,17 +353,20 @@ func _on_input_event(_viewport: Node, event: InputEvent, _shape_idx: int):
 			if I.active:
 				on_item_used(I.active)
 			else:
-				handle_command(event_index)
-				
-				times_clicked += 1
+				if event.double_click:
```

Comment:
To accommodate touchscreen devices, it would be best to create a method in **PopochiuUtils** that checks for double clicks or double taps. I suggest something like this:
```
static func is_double_click_or_tap(event: InputEvent) -> bool:
    if is_click_or_touch_pressed(event):
        if event is InputEventMouseButton:
            return event.double_click
        elif event is InputEventScreenTouch:
            return event.double_tap
    return false
```